liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/LooseEarApplication.java

```diff
@@ -90,6 +91,17 @@ public Element addWarModule(MavenProject proj, Artifact artifact, File warSource
         Element warArchive = config.addArchive(getModuleUri(artifact));
         config.addDir(warArchive, warSourceDir, "/");
         config.addDir(warArchive, new File(proj.getBuild().getOutputDirectory()), "/WEB-INF/classes");
+        
+        
+        // retrieve the directories defined as resources in the maven war plugin
+        Map<String,String> webResources = MavenProjectUtil.getWebResourcesConfiguration(proj);
+        if (webResources != null) {
+            for (String directory : webResources.keySet()) {
+                String targetPath = webResources.get(directory)==null ? "/" : "/"+webResources.get(directory);
```

Comment:
`get` should not return null since this is iterated over the `keySet`.  Is this just an extra sanity check / for coding safey?

Reply:
If you look at `io.openliberty.tools.maven.utils.MavenProjectUtil.getWebResourcesConfiguration(MavenProject)`  it's building a map of the dir => targetPath, so it will legitimately put a 'null' value into the map.

Also.. I purposely copied line for line from DeployMojoSupport  to make this easier to identify in the future.

For the prototype in https://github.com/OpenLiberty/ci.maven/issues/1104 I did go ahead and refactor this into something that'd be more reusable as a "utility" method....but that wasn't trivial and probably should be done with more care.  

FWIW, let me mention that the paths at the moment don't work as well on Windows.   IIRC, you can't use a relative path on Windows..and my prototype also switches to use:   java.nio.file.Path to fix this.